Tests/ReactiveSwiftTests/SignalProducerLiftingSpec.swift

```diff
@@ -1666,5 +1666,47 @@ class SignalProducerLiftingSpec: QuickSpec {
 				expect(latestValues?.1) == 2
 			}
 		}
+		
+		describe("groupBy") {
```

Comment:
I think asserting just the specific event is enough. The `Signal` contract guarantees only one would ever be sent. If multiple terminal events are sent, that is a fault in the basics and should have been caught somewhere else.